partiql-planner/src/main/kotlin/org/partiql/planner/internal/PlannerFlag.kt

```diff
@@ -0,0 +1,20 @@
+package org.partiql.planner.internal
+
+internal enum class PlannerFlag {
+    /**
+     * Determine the planner behavior upon encounter an operation that always returns MISSING.
+     *
+     * If this flag is included:
+     *
+     *    The problematic operation will be tracked in problem callback as a error.
+     *
+     *    The result plan will turn the problematic operation into an error node.
+     *
+     * Otherwise:
+     *
+     *    The problematic operation will be tracked in problem callback as a missing.
+     *
+     *    The result plan will turn the problematic operation into a missing node.
+     */
+    SIGNAL_MODE
```

Comment:
This is ok and we don't have to change anything because it is internal.

We are introducing two concepts, compiler _warnings_ and a warning for "always missing". What we are calling "signal" is really emit warnings for always missing + treat warnings as errors -> resulting in a throw